### PR TITLE
Also load .uc.js, .uc.xul files, and userChrome.xul

### PR DESCRIPTION
**Pull request by Sporif**
Instead of loading all scripts in one file, you can separate them to keep some order. Now you can just copy scripts from repositories like [this](https://github.com/Endor8/userChrome.js) to the chrome folder.

I got it from [here](http://mozilla.zeniko.ch/userchrome.js.html) (Sub-Script/XUL Loader), but made small changes to load userChrome.js as well (regex on line 23) and work with future Firefox versions (nsILocalFile > nsIFile on line 16).

### CODE REVIEW

File: userChrome.xml
Comment (nuchi) [inline userChrome.xml:15]: Please split up `foo(...).bar(...).baz(...)...` into multiple lines:
```javascript
foo(...)
    .bar(...)
    .baz(...);
```
Please also use spaces over tabs, and consistent indentation. (2 or 4 spaces, I don't care which)

**Review by nuchi — CHANGES_REQUESTED** [review]
Thanks for opening a PR! I like the changes. a few things before I can merge (and one nitpick left as an in-line comment):
1. Please check whether the author has given permission for their code to be copied/distributed
2. If so, please check the terms of the license under which the author gives permission for copying/distribution, and ensure that your PR follows the terms of that license. If you do that, I'll check whether that license is compatible with the MIT license.
3. Please update the README as well to indicate to readers that they may also include their own `*.uc.js` and `*.uc.xul` files.

I'll merge after that. Thanks!

**Comment by Sporif** [comment]
I can't find any permissions for the code to be copied or distributed. It's just linked as a data uri, no license or anything. I don't even know how to contact the author. What should I do then?

**Comment by nuchi** [comment]
Good question. I don't know. You could try to find the author, I guess. I don't want to ignore the licensing issues, so if you can't locate the author then I won't merge. If you want to host the fork at your own risk, that's up to you of course.

**Comment by nuchi** [comment]
Alternatively, you could take careful notes on how the script works, then hide the original, and then recreate it from your notes. Then the recreation would be yours.

**Comment by Sporif** [comment]
I need to think about this, I'll close the pull request for now.

**Comment by nuchi** [comment]
No worries

**Pull request closed without merging**
